package.json

```diff
@@ -23,7 +23,7 @@
     "method-override": "2.3.3",
     "moment": "2.10.3",
     "passport": "0.2.2",
-    "passport-saml": "0.5.3",
```

Comment:
Hi,
Any update on this?  I've a blog post ready to go just waiting for the saml
service to be updated first.

Regards,
Philip

On Tue, Feb 23, 2016 at 10:32 AM, Jason Madigan <notifications@github.com>
wrote:

> In package.json
> <https://github.com/feedhenry-templates/saml-service/pull/13#discussion_r53795997>
> :
>
> > @@ -23,7 +23,7 @@
> >      "method-override": "2.3.3",
> >      "moment": "2.10.3",
> >      "passport": "0.2.2",
> > -    "passport-saml": "0.5.3",
>
> Not readily available, but there is one in the Engineering AWS account.
> I'll spin it up and give this a spin later this week (you may need to prod
> me if I forget).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/feedhenry-templates/saml-service/pull/13/files#r53795997>
> .
>